tests/query.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2016 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+extern crate time;
+
+extern crate mentat;
+extern crate mentat_core;
+extern crate mentat_db;
+
+use mentat_core::{
+    TypedValue,
+    ValueType,
+};
+
+use mentat::{
+    NamespacedKeyword,
+    QueryResults,
+    new_connection,
+    q_once,
+};
+
+#[test]
+fn test_query() {
+    let mut c = new_connection("").expect("Couldn't open conn.");
+    let db = mentat_db::db::ensure_current_version(&mut c).expect("Couldn't open DB.");
+
+    // Rel.
+    let start = time::PreciseTime::now();
+    let results = q_once(&c, &db.schema,
+                         "[:find ?x ?ident :where [?x :db/ident ?ident]]", None)
+        .expect("Query failed");
+    let end = time::PreciseTime::now();
+
+    // This will need to change each time we add a default ident.
+    assert_eq!(37, results.len());
```

Comment:
Hmm.  You could export something from `bootstrap` to make this a little future proof, but that's a weird thing to leak to the global namespace.